jobs/build-triggers/tendrl-build-triggers-pr-builds.yml

```diff
@@ -0,0 +1,261 @@
+            "find ${{rwd}}/rpms/pr/{package_prefix}{package_name}/${{branch}} \
+            -type f ! -iname '*.rpm' -delete"
+          ssh root@${{build_server}} \
+            "find ${{rwd}}/rpms/pr/{package_prefix}{package_name}/${{branch}} \
+            -type f -iname '*.src.rpm' -delete"
+
+          # prepare rpm repo
+          ssh root@${{build_server}} \
+            "createrepo ${{rwd}}/rpms/pr/{package_prefix}{package_name}/${{branch}}/"
+
+          # prepare repofile
+          repo_file="${{rwd}}/rpms/pr/{package_prefix}{package_name}/${{branch}}/{package_prefix}{package_name}-${{branch}}.repo"
+          ssh root@${{build_server}} \
+            "echo '[{package_prefix}{package_name}-${{branch}}]' > ${{repo_file}} ;\
+            echo 'name={package_prefix}{package_name}-${{branch}}' >> ${{repo_file}} ;\
+            echo 'baseurl=http://artifacts.ci.centos.org/tendrl/pr/{package_prefix}{package_name}/${{branch}}/' >> ${{repo_file}} ;\
+            echo 'skip_if_unavailable=True' >> ${{repo_file}} ;\
+            echo 'gpgcheck=1' >> ${{repo_file}} ;\
+            echo 'gpgkey=https://copr-be.cloud.fedoraproject.org/results/tendrl/${{TARGET_REPO}}/pubkey.gpg' >> ${{repo_file}} ;\
+            echo 'repo_gpgcheck=0' >> ${{repo_file}} ;\
+            echo 'enabled=1' >> ${{repo_file}} ;\
+            echo '' >> ${{repo_file}};"
+
+          # upload generated repo to artifacts.ci.centos.org
+          set +x
+          key=$(cat ~/duffy.key)
+          ssh root@${{build_server}} \
+            "RSYNC_PASSWORD=${{key:0:13}} rsync -av \
+            ${{rwd}}/rpms/pr/ \
+            tendrl@artifacts.ci.centos.org::tendrl/pr/"
+          # cleanup older packages for the particular PR
+          ssh root@${{build_server}} \
+            "RSYNC_PASSWORD=${{key:0:13}} rsync -av --delete \
+            ${{rwd}}/rpms/pr/{package_prefix}{package_name}/${{branch}}/ \
+            tendrl@artifacts.ci.centos.org::tendrl/pr/{package_prefix}{package_name}/${{branch}}/"
+          set +x
+
+          echo
+          echo "##############################################################"
+          echo "REPO: http://artifacts.ci.centos.org/tendrl/pr/{package_prefix}{package_name}/${{branch}}/{package_prefix}{package_name}-${{branch}}.repo"
+          echo "##############################################################"
+          echo
+
+    publishers:
+      - trigger-parameterized-builds:
+          - project: 'tendrl-build-X-build_server-{project_name}-teardown'
+            property-file: env.sh
+            condition: ALWAYS
+
+      - email:
+          recipients: dahorak@redhat.com sds-mgmt-dev@redhat.com
+          send-to-individuals: false
+            
+
+- project:
+    name: 'PR'
+    project_name: 'PR'
+    target_repo: 'Features'
+    copr_config: '~/.copr-tendrl.conf'
+    copr_user: 'tendrl'
+    triggers:
+      - github-pull-request:
+          admin-list:
+            - dahorak
+          white-list:
```

Comment:
Is there a way to input this rather than hardcoding?

Reply:
@nthomas-redhat it should be possible, to automatically fetch the list of people from the organization specified below in the `org-list` section, but there seems to be some problem with this and I'm not sure why it doesn't work and where might be the problem with that.
I'll try to check that once more and find some more information about that - that will be definitely the best solution.

Reply:
@nthomas-redhat I've found partial solution for this: it works for people with Public membership.

@r0h4n could you please check, if *centos-ci* user is part of Tendrl organization?
That might solve the problem with people with private membership in Tendrl org.
(I know, you already added it https://github.com/Tendrl/tendrl-ci/issues/6#issuecomment-362198957, but it seems like it is not there https://github.com/orgs/Tendrl/people...)